frontend/app/routes/protected/multi-channel/send-validation.tsx

```diff
@@ -68,6 +68,13 @@ export async function loader({ context, request }: Route.LoaderArgs) {
   // TODO: the id will likely come from a path param in the URL?
   const personSinCase = await getSinCaseService().getSinCaseById('000000000');
 
+  if (personSinCase === undefined) {
+    throw new Response(JSON.stringify({ status: 404, message: 'Case not found' }), {
+      status: 404,
+      headers: { 'Content-Type': 'application/json' },
+    });
```

Comment:
You can remove the `headers` prop here since the framework should handle the content type for you (when calling a loader).

Also for the status, you can use `status: HttpStatusCodes.NOT_FOUND` which is marginally easier to read.